source/templates/code_signing.slim

```diff
@@ -88,20 +88,40 @@
 				article
 					aside.sidebar
 						.menu
-							button.menu-option[ng-repeat="menu in provProfileCtrl.menus" ng-click="provProfileCtrl.selectedMenu = menu" ng-class="{'selected': menu == provProfileCtrl.selectedMenu}" ng-bind="menu.title"]
+							button.menu-option[ng-repeat="menu in provProfileCtrl.menus" ng-click="provProfileCtrl.selectedMenu = menu" ng-class="{'selected': menu == provProfileCtrl.selectedMenu}"]
+								span.title[ng-bind="menu.title"]
+								span.status[ng-if="menu.cssClass == 'matching-certificates' || menu.cssClass == 'devices'" ng-class="menu.validationStatus()"] = svg("dot")
```

Comment:
It might be prettier to do this with display: none. And add the `menu.cssClass` in `button.menu-option`'s class attribute (ng-class would be better, but that is already used in "object" mode, and you can't mix that with this other one).

Then, in CSS do something like `.menu-option:not(.matching-certificates) .status { display: none }`, etc.